packages/m365-admin/package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "@uifabric/dashboard-grid-layout",
```

Comment:
We really shouldn't have anything Microsoft product specific as a Fabric package. If this is to be an internal package, we should put this inside a VSTS repo (there's a precedence for that)

Reply:
I don't believe these are Microsoft specific. I believe the intention is to have a collection of generalized components who's purpose is admin interfaces rather than applications. One of primary goals of keeping them in this repo is discoverability and collaboration. We are going to work to make sure these components leverage Fabric as much as possible and follow consistent conventions.

Reply:
I think you're mistaking what I'm getting at here. I don't think we should publish package names that are Microsoft product specific.

Reply:
We can rename the package if that's better. The controls will not have anything Microsoft specific, but more like dashboard specific for a portal home page.